### ISSUE TITLE
About the difference of pretrained model

### ISSUE THREAD

**Issue by r-matsuzaka** [MouseLand/cellpose#400]
Hi.
I have a question about pretrained model.
When I download the pretrained models.
There are many files.
Are there any description about this?

If it is not available, I want to know at least

1. the difference between cytotorch and cyto2torch
2. meaning of suffix like _0, _1 and _3. Larger is better?

**Comment by kevinjohncutler** [comment]
Good questions, I've been wondering about one of these points myself. The _0,_1,_2,_3 models are trained differently (@carsen-stringer can tell us more about that, I am not sure exactly - maybe just partitioning or shuffling the datasets with a different seed) with the goal to average the slightly different network outputs to give more robust flow and cell probability. Some models only have _0, such as my Omnipose ones, because we haven't done the additional training to do that averaging. There are also different models for pytorch vs mxnet trained on the same data as well as models trained on different datasets, such as my bact models vs cyto and cyto2 (bact is just phase contrast bacteria, cyto is the original Cellpose model, cyto2 is trained on the cyto dataset plus user-submitted data).

**Comment by carsen-stringer** [comment]
the only difference between `_0`, `_1` ... is the random seed for the weight initialization, otherwise the image augmentations etc are the same. We find some improvement from averaging the outputs from these models and so include all of them for the user. by default the averaging is on.

**Comment by r-matsuzaka** [comment]
Thank you  so much!
